test/IRGen/arm_to_thumb_darwin.sil

```diff
@@ -0,0 +1,28 @@
+// RUN: %swift -target armv7-apple-ios7 %s -gnone -emit-ir -o - | %FileCheck %s -check-prefix=IOS
+// RUN: %swift -target armv7k-apple-watchos2 %s -gnone -emit-ir -o - | %FileCheck %s -check-prefix=WATCHOS
+
+// REQUIRES: CODEGENERATOR=ARM
```

Comment:
Oh, I missed that it was `%swift` rather than `%target-build-swift`. Sorry for the noise.